src/tools/Reporting/Reporting/Reporter.cs

```diff
@@ -67,7 +67,7 @@ private void Init()
             foreach (var kvp in configs.Split(';'))
             {
                 var split = kvp.Split('=');
-                run.Configurations.Add(split[0], split[1]);
+                run.Configurations.Add(split[0], split[1].ToLower());
```

Comment:
I am not exactly sure of what the impact of this would be. I think this should get split out into another PR so we have more time to think about it.